tests/roles/dataplane_adoption/tasks/cinder_verify.yaml

```diff
@@ -4,22 +4,26 @@
     osc_header: |
       alias openstack="oc exec -t openstackclient -- openstack"
 
-- name: Verify that the volumes have correct name, status and size
-  when: prelaunch_test_instance|bool
+- name: Verify that the volumes and snapshots have correct name, status and size
```

Comment:
hmm, these validations doesn't seem to be running, did we miss something anywhere?
this is the adoption-standalone-to-crc-ceph job

TASK [dataplane_adoption : Verify that the volumes and snapshots have correct name, status and size] ***
skipping: [localhost] => {"changed": false, "false_condition": "cinder_volume_backend in supported_volume_backends", "skip_reason": "Conditional result was False"}

TASK [dataplane_adoption : verify the backup of disk volume] *******************
skipping: [localhost] => {"changed": false, "false_condition": "cinder_backup_backend in supported_backup_backends", "skip_reason": "Conditional result was False"}

Reply:
yea, the ceph job got
```
export CINDER_VOLUME_BACKEND_CONFIGURED=\"false\"
export CINDER_BACKUP_BACKEND_CONFIGURED=\"false\"
```

Reply:
zuul provided [inventory](https://logserver.rdoproject.org/58/658/4551da92f65083ba6899ec1a67c24ff17f0bf107/github-check/adoption-standalone-to-crc-ceph/b29ade9/zuul-info/inventory.yaml) has no supported backends defined from this dependency https://review.rdoproject.org/r/c/rdo-jobs/+/55084/2/zuul.d/_data_plane_adoption.yaml#34

not sure why